### PR TITLE
Integrate ConfD

### PR DESCRIPTION
**Pull request by eldadcohen1**
#380 
Integrate confD.
Chanage Docker file
Create config in assets

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Declare files that will always have LF line endings on checkout.
+*.sh text eol=lf
+env.list text eol=lf
```
Comment (asafMasa) [inline .gitattributes]: add new line

File: confd/production.tmpl [hunk 2]
```diff
@@ -0,0 +1,127 @@
+window.config = Object.freeze({
```
Comment (asafMasa) [inline confd/production.tmpl]: fix from js to json

File: confd/production.toml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[template]
+src = "production.tmpl"
+dest = "/opt/ansyn/app/assets/config/app.config.module.js"
```
Comment (asafMasa) [inline confd/production.toml]: change app.config.module.js to app.config.json

File: src/app/app-routing.module.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { Component, HostBinding, NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { AnsynComponent } from './ansyn/ansyn.component';
-import { configuration } from '../configuration/configuration';
+import { AppSettings } from './app-models/settings';
```
Comment (asafMasa) [inline src/app/app-routing.module.ts]: remove

File: src/app/app.module.ts [hunk 5]
```diff
@@ -25,6 +24,9 @@ import { OpenLayerVisualizersModule, OpenLayersVisualizerMapType } from '@ansyn/
 import { ContextElasticSource } from '@ansyn/context/';
 import { ContextProxySource } from '@ansyn/context';
 import { ContextEntityVisualizer } from './app-visualizers/context-entity.visualizer';
+import { AppSettings } from './app-models/settings';
```
Comment (asafMasa) [inline src/app/app.module.ts]: ?

File: src/assets/config/app.config.json [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+                "datum": "wgs84",
+                "projection": "geo"
+            },
+            "to": {
+                "datum": "ed50",
+                "projection": "utm"
+            }
+        }
+    }
+}
```
Comment (asafMasa) [inline src/assets/config/app.config.json]: line

File: src/configuration/configuration.dev.ts [hunk 7]
```diff
@@ -6,95 +6,12 @@
 import { defaultCase } from './cases/default_case';
 
 export const configuration = {
-	env: 'dev',
+	env: 'default',
```
Comment (asafMasa) [inline src/configuration/configuration.dev.ts]: dev

File: src/main.ts [hunk 8]
```diff
@@ -1,14 +1,70 @@
...
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
 import { configuration } from './configuration/configuration';
+import { ContextElasticSource,ContextProxySource } from "@ansyn/context";
 
-if (configuration.production) {
-  enableProdMode();
-}
+// if (configuration.production) {
```
Comment (asafMasa) [inline src/main.ts]: delete
space

File: src/typings.d.ts [hunk 9]
```diff
@@ -8,3 +8,5 @@ declare module "*.json" {
 	const value: any;
 	export default value;
 }
+
+declare var config: any;
```
Comment (asafMasa) [inline src/typings.d.ts]: remove

File: .gitattributes (hunk 1 above)
Comment (eldadcohen1) [reply .gitattributes]: done

File: confd/production.tmpl (hunk 2 above)
Comment (eldadcohen1) [reply confd/production.tmpl]: done

File: src/app/app-routing.module.ts (hunk 4 above)
Comment (eldadcohen1) [reply src/app/app-routing.module.ts]: done

File: src/app/app.module.ts (hunk 5 above)
Comment (eldadcohen1) [reply src/app/app.module.ts]: done

File: src/configuration/configuration.dev.ts (hunk 7 above)
Comment (eldadcohen1) [reply src/configuration/configuration.dev.ts]: done

File: src/main.ts (hunk 8 above)
Comment (eldadcohen1) [reply src/main.ts]: done

File: src/typings.d.ts (hunk 9 above)
Comment (eldadcohen1) [reply src/typings.d.ts]: done

**Pull request merged**